inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -1011,16 +1005,26 @@ def normalize_affiliations(obj, eng):
                 continue
             matched_author_affiliations = _match_lit_author_affiliation(raw_aff)
             if matched_author_affiliations:
-                assigned_affiliations = _assign_matched_affiliations_to_author(
-                    author_affiliations, matched_author_affiliations
-                )
-                matched_affiliations[raw_aff] = assigned_affiliations
+                _assign_institution_reference_to_incomplete_affiliations(matched_author_affiliations)
```

Comment:
You don't need to do this here as you'll be doing it in the next workflow step anyway